backend/backend/config.py

```diff
@@ -3,6 +3,10 @@
 Use env var to override
 """
 import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
```

Comment:
This should happen automagically, but maybe not if running outside of flask app

Reply:
ah i see. yea for debugging in sometimes run outside of flask